src/model/pwrmgmt.go

```diff
@@ -23,6 +24,8 @@ var scalProfilesMap = map[string]ScalProfiles{
 type CpuGovernanceRule struct {
 	CPUs    string `yaml:"cpus"`
 	ScalGov string `yaml:"scaling_governor"`
+	MinFreq string `yaml:"min_frequency"`
+	MaxFreq string `yaml:"max_frequency"`
```

Comment:
```suggestion
	MinFreq string `yaml:"min_freq"`
	MaxFreq string `yaml:"max_freq"`
```
Freq is a well-known short form for frequency.